xchange-huobi/src/main/java/org/knowm/xchange/huobi/dto/account/results/HuobiWithdrawFeeRangeResult.java

```diff
@@ -0,0 +1,17 @@
+package org.knowm.xchange.huobi.dto.account.results;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.knowm.xchange.huobi.dto.HuobiResult;
+import org.knowm.xchange.huobi.dto.account.HuobiWithdrawFeeRange;
+
+public class HuobiWithdrawFeeRangeResult extends HuobiResult<HuobiWithdrawFeeRange> {
```

Comment:
you can just use `HuobiResult<HuobiWithdrawFeeRange>` directly

Reply:
Done